app/_src/networking/transparent-proxy/configuration-on-kubernetes.md

```diff
@@ -0,0 +1,421 @@
+---
+title: Adjusting Transparent Proxy Configuration on Kubernetes
+content_type: how-to
+---
+
+{% assign docs = "/docs/" | append: page.version %}
+{% assign Kuma = site.mesh_product_name %}
+{% assign kuma-system = site.mesh_namespace %}
+{% assign controlPlane = site.set_flag_values_prefix | append: "controlPlane" %}
+{% assign transparentProxy = site.set_flag_values_prefix | append: "transparentProxy" %}
+{% assign kuma-control-plane = site.mesh_cp_name %}
+{% assign kuma-control-plane-config = kuma-control-plane | append: "-config" %}
+
+{% assign configuration-in-configmap = "Configuration in ConfigMap <sup>(experimental)</sup>" %}
+
+The default transparent proxy configuration works well for most scenarios, but there are cases where adjustments are needed. This guide explains the various methods available for modifying the configuration, along with their limitations and recommendations on when to use each one. While these methods can be used individually, they can also be combined when necessary.
+
+{% warning %}
+It’s best to stick to one method whenever possible. Using more than one can make things more complicated and harder to troubleshoot, as it may not be clear where each setting comes from. If you need to combine methods, check the [**Order of Precedence**](#order-of-precedence) to see what the final configuration will look like based on the priority of each setting.
```

Comment:
```suggestion
It’s best to stick to one method whenever possible. Using more than one can make things more complicated and harder to troubleshoot, as it may not be clear where each setting comes from. If you need to combine methods, check the [**Order of Precedence section**](#order-of-precedence) to see what the final configuration will look like based on the priority of each setting.
```

Reply:
addressed